src/grc65/main.c

```diff
@@ -607,15 +610,14 @@ static void DoHeader (void)
             "\t.byte %i\n"
             "\t.byte %i\n"
             "\t.byte %i, %i, %i, %i, %i\n\n"
-            "\t.word 0\n"
+            "\t.word ((__VLIR0_LAST__ - __VLIR0_START__ - __BSS_SIZE__) + 253) / 254\n"   /* length in blocks */
```

Comment:
253 and 254 as a result of the two link bytes? Please use geoConvert converted files as reference.
By the way next file(1) will support CVT :-)

Reply:
There is no way to use geoConvert files as a reference - unless you can provide a pair of those and the corresponding ca65 source to produce them.

What i am doing instead is writing them to disk, and then "validate" in the GEOS menu - if *that* doesn't produce the correct block numbers, then everything is broken beyond repair (in GEOS)

And yes of course, it is rounding up to the next full block. But i must be missing something in the calculation

Reply:
Indeed, we have no source of Desktop to see validate's implementation.